_scripts/check-output/main.go

```diff
@@ -0,0 +1,109 @@
+package main
```

Comment:
Don't we want to put this functionality to the `steps-check` step?

Reply:
I think it is more like our `simple-git-clone` step, a utility Step we can use in the Step e2e test cases.
I suggest moving it to a separate repository.

Reply:
https://github.com/bitrise-steplib/bitrise-step-check-step-outputs